src/web/app/pages-session/session-submission-page/session-submission-page.component.html

```diff
@@ -35,7 +35,7 @@ <h4>Previewing Session as
     </div>
   </div>
 </div>
-<div class="row" *ngIf="feedbackSessionSubmissionStatus === FeedbackSessionSubmissionStatus.OPEN">
+<div class="row mobile-invisible" *ngIf="feedbackSessionSubmissionStatus === FeedbackSessionSubmissionStatus.OPEN">
```

Comment:
Consider the second condition: feedbackSessionSubmissionStatus === FeedbackSessionSubmissionStatus.OPEN. Try opening a feedback submission that has closed: The prompt you generated still shows up, which is confusing because users can't actually resubmit since the submission has closed.
You can add this condition check, which should work, though it'd be even better if you can find some way to tie the prompt to this specific place (so there's only 1 place to change if a new condition needs to be added)

Reply:
Actually a good idea. It seems that I shall do a full run on this change in different states of the submission process. 👍 Will first bind it to the condition.